app/dashboard/models.py

```diff
@@ -70,6 +70,13 @@ def exclude_by_status(self, excluded_statuses=None):
 
         return self.exclude(idx_status__in=excluded_statuses)
 
+    def filter_by_status(self, filtered_status=None):
+        """Filter results with a status matching the provided list."""
+        if filtered_status is None:
```

Comment:
I'm going to add a check in here on `isinstance()` when `filtered_status is not None` to validate we're receiving a list.

Reply:
What should be returned here if `filtered_status` is not a list?

Reply:
@mbeacom Everything else that you suggested is fixed. This is the last bit that needs to be taken care of. Please let me know :)